Description: fix the fact that wayland does not recognise the running app
Author: Ben Burton <bab@debian.org>
Origin: upstream
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- regina-normal-7.3.1/qtui/src/org.computop.Regina.desktop
+++ regina-normal-7.3.1/qtui/src/org.computop.Regina.desktop
@@ -6,5 +6,6 @@ GenericName=3-Manifold Topology
 Comment=Mathematical software for low-dimensional topology
 Icon=regina
 Exec=regina-gui %F
+StartupWMClass=io.github.regina-normal.regina-gui
 Categories=Math;Education;Science;Qt;
 MimeType=application/x-regina;
